roles/mongodb/tasks/users.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: users > Create Mongo users
+  mongodb_user:
+    name: "{{ item.name }}"
+    password: "{{ item.password|default(omit) }}"
+    database: "{{ item.database }}"
+    roles: "{{ item.roles|default(omit) }}"
+    state: "{{ item.state|default(omit) }}"
+    login_database: "{{ item.login_database|default(omit) }}"
+    login_host: "{{ item.login_host|default('localhost') }}"
+    login_password: "{{ item.login_password|default(omit) }}"
+    login_port: "{{ item.login_port|default('27017') }}"
```

Comment:
```suggestion
    login_port: "{{ item.login_port|default(omit) }}"
```

Reply:
I kept default values from the ansible module, so it's easy to read without searching documentation